TypeCobol/Compiler/CupCommon/CobolWordsTypes.cs

```diff
@@ -21,6 +22,37 @@ public class PairTokenList : List<Tuple<Token,Token>> { };
     /// </summary>
     public class PairTokenListList : List<Tuple<List<Token>, List<Token>>> { };
```

Comment:
Is it still required with the introduction of the new `CupReplaceOperation` class ?